javascript/exercise_form/signUp.js

```diff
@@ -0,0 +1,189 @@
+const email = document.querySelector('.email');
+const username = document.querySelector('.username');
+const password = document.querySelector('.password');
+const confirmPassword = document.querySelector('.confirm-password');
+const signUpForm = document.getElementById('sign-up-form');
+let isError = false;
+
+/**
+  * @description Show and hide error
+  *
+  * @param {Object} element of input
+  */
+function showHideErrors(element) {
```

Comment:
give a param for `isError` to this function will better to use global variable checking